doc/HttpLuaModule.wiki

```diff
@@ -658,6 +657,21 @@ Within external script files, PCRE sequences presented as long-bracketed Lua str
     -- evaluates to "1234"
 </geshi>
 
+As noted earlier, PCRE sequences presented within <code>*_by_lua_block {}</code> directives, available following the <code>v0.9.17</code> release, do not require modification.
```

Comment:
nitpick++: I'd use dashes (`- thing -`), or maybe even parenthesis instead of comma here, to not lose context. Very much a nitpick.